config/routes.rb

```diff
@@ -116,7 +116,7 @@
     get "finding_aids" => :index
     get "finding_aids/:id" => :show
     get "finding-aids/:id" => :show
-    get "finding_aids.json", to: redirect("assets/finding_aids.json")
```

Comment:
@sensei100 @cdoyle-temple just an fyi that we didn't make the assets folder writable just /app/public/cache, /tmp, /app/log and app/tmp

Reply:
@dkinzer I think it should be okay because he removed that line.